doc/intensity-recipe.yaml

```diff
@@ -0,0 +1,1277 @@
+Kinematics:
```

Comment:
The copy is necessary because of the relative path `"../intensity-recipe.yaml"`. You want this to work from both the `examples/jupyter` directory and the `doc/usage` directory.

Reply:
But cant you copy this file when building the docs only, I dont like this being in the version control twice...

Reply:
Ahh okay sorry my bad, yes, this file was committed by accident. The recipe file is already copied correctly though
https://github.com/ComPWA/tensorwaves/pull/65/files#diff-684a0ceb2878d48b19612ca360e277efR37

Reply:
fa46994b93ff1a3e45c8459b052819e436e0a2b7

Reply:
Ok